operators/go/gen_subm_operator.sh

```diff
@@ -106,7 +106,7 @@ function add_subm_engine_to_operator() {
 
   operator-sdk add controller --api-version=$api_version --kind=$kind
 
-  controller_file_src=$op_gen_dir/submariner_controller.go
+  controller_file_src=$op_gen_dir/submariner_controller.go.spec
```

Comment:
Sure, whatever suffix will work. Maybe even ".source" or some other suffix would be even clearer that this is not a part of the code but is used for the generated code.

I used .spec since that's what you use in packaging nomenclature, but it doesn't really have any significance.